pudb/debugger.py

```diff
@@ -342,6 +342,22 @@ def set_frame_index(self, index):
 
         self.ui.stack_list._w.set_focus(self.ui.translate_ui_stack_index(index))
 
+    def open_file_to_edit(self, index):
+        self.curframe, line_number = self.stack[index]
+        filename = self.curframe.f_code.co_filename
+
+        editor = os.environ.get("EDITOR", "nano")
+        editor = editor.lower()
+
+        editor_command = f"{editor}"
+        if editor in {"vi", "vim", "nvim", "emacs"}:
+            editor_command = f"{editor} +{line_number}"
+        elif editor == "nano":
+            editor_command = f"{editor} -l {line_number}"
```

Comment:
According to `man nano` at least on my machine, nano also supports the `+line_number` syntax. Maybe we should just always do this regardless of the editor. I suspect other editors like vscode will probably also support this.

Reply:
DONE